ConsentViewController/Classes/SPConsentManager.swift

```diff
@@ -17,7 +17,7 @@ import UIKit
     /// By default, the SDK will remove all user consent data from UserDefaults, possibly triggering a message to be displayed again next time
     /// `.loadMessage` is called.
     /// Set this flag to `false` if you wish to opt-out from this behaviour.
```

Comment:
@Nevazhnovu can you update the comment to reflect the new behaviour?

Reply:
@andresilveirah fixed, as well as `README`